### ISSUE TITLE
InterfaceWithCleaver2, part 2

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#1744]
- [ ] Support double data on input fields
- [ ] Investigate padding
- [ ] Support for cell-centered inputs

**Comment by dcwhite** [comment]
@jessdtate Not sure how relevant these parameters are now. Do we need a meeting on this one?

**Comment by jessdtate** [comment]
I think these still need to be address.  We can meet if you want.

**Comment by dcwhite** [comment]
Could @allywarner work on this one?

**Comment by jessdtate** [comment]
Yeah, this is something @allywarner can work on.  We can meet to talk this.

**Comment by dcwhite** [comment]
@jessdtate says to profile the IBBM cleaver example.

**Comment by dcwhite** [comment]
**Analysis (2026-06-12)**

Checklist/tracking issue for Cleaver2 enhancements. Items include: double-precision input support, cell-centered field inputs, background mesh timing improvements, and padding removal. The padding removal was noted as already done.

No new code investigation needed — this is a project management tracker. Suggest reviewing the checklist items against current master to update which are complete before the next sprint.
